sixtyfps_compiler/widgets/common/common.60

```diff
@@ -84,4 +84,27 @@ export TextEdit := ScrollView {
             }
         }
     }
-}
\ No newline at end of file
+}
+
+export AboutSixtyFPS := Rectangle {
+    VerticalLayout {
+        padding: 12px;
+        spacing: 8px;
+        alignment: start;
+        Text {
+            text: "Made width";
+            font-size: 24px;
+            font-weight: 700;
+            horizontal-alignment: center;
+        }
+        Image {
+            source: @image-url("sixtyfps_logo.svg");
+            preferred-width: 128px;
+        }
+        Text {
+            text: "Version 0.1.3\n© 2021 SixtyFPS GmbH\nhttps://sixtyfps.io/";
```

Comment:
Not sure about the copyright there.  It might look like the all app has this copyright

Reply:
Okay, I'll remove it